en/flying/led_meanings.md

```diff
@@ -0,0 +1,38 @@
+# LED meanings
+
+[Pixhawk flight controllers](../flight_controller/pixhawk_series.md) have an RGB LED to indicate the current status of the FMU.
+
+![LED meanings](../../images/led_meanings.gif)
+
+> **Note:** **GPS unit vs GPS Lock** 
+Please note that Pixhawk’s RGB LED will only turn green and allow you to execute guided mission only if you have a GPS unit and Pixhawk is able to validate the global position from the GPS unit. If you have a GPS unit plugged in but the RGB LED continues to be blue, verify that Pixhawk is reading your GPS properly, and that your GPS is sending a proper GPS position.
```

Comment:
Grammatical errors. Perhaps:

> The LED will only turn green (ready to execute guided mission) if PX4 is able to validate the global position. If the LED does not change from blue to green, verify that the GPS module is properly attached, Pixhawk is reading your GPS properly, and that the GPS is sending a proper GPS position.

Also note for above statement - *how* would someone verify this?

Reply:
maybe with QGC's GPS status?

Reply:
Sorry, ignore my other note. Yes this would work at least for knowing that you're getting signal. Perhaps update with  `(using QGroundControl).` at the end. Even better if there is a good QGC link to use (probably not).